packages/cli/templates/app/app/pages/index.tsx

```diff
@@ -3,27 +3,22 @@ import {Head} from '@blitzjs/core'
 const Home = () => (
   <div className="container">
     <Head>
-      <title><%= name %></title>
+      <title>yep</title>
```

Comment:
Oops, this was my bad

```suggestion
      <title><%= name %></title>
```